src/decoder/cuda-decoder-utils.h

```diff
@@ -0,0 +1,431 @@
+// decoder/cuda-decoder-utils.h
+
+// Copyright      2018  Zhehuai Chen
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http:// www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef KALDI_CUDA_DECODER_UTILS_H_
+#define KALDI_CUDA_DECODER_UTILS_H_
+
+
+#include <cuda_runtime.h>
```

Comment:
Please clean this up. cuda_runtime.h and cuda_runtime_api.h are both included twice. Also, do we need math_constants.h?

Reply:
done.